pkg/plugins/runtime/k8s/controllers/mesh_defaults_controller_test.go

```diff
@@ -42,7 +43,9 @@ var _ = Describe("MeshDefaultsReconciler", func() {
 		store, err := k8s.NewStore(kubeClient, k8sClientScheme, k8s.NewSimpleConverter())
 		Expect(err).ToNot(HaveOccurred())
 
-		secretStore, err := secrets_k8s.NewStore(kubeClient, kubeClient, "default")
+		scheme, err := bootstrap_k8s.NewScheme()
```

Comment:
we need to bring in the actual scheme we're using so that the Mesh CRD can be hooked up as owner. Otherwise we will get:

```
no kind is registered for the type v1alpha1.Mesh in scheme
```